op-guide/tune-tikv.md

```diff
@@ -26,21 +26,233 @@ Each CF also has a separate `write buffer`. You can configure the size by settin
+# wal-ttl-seconds = 0
+# wal-size-limit = 0
+
+# In most cases, set the maximum total size of RocksDB WAL logs to the default value.
+# max-total-wal-size = "4GB"
+
+# This parameter is for enabling or disabling the statistics information of RocksDB.
+# enable-statistics = true
+
+# This parameter is for enabling the readahead feature during RocksDB compression. If you are using mechanical disks, it is recommended to set the value to 2MB at least.
+# compaction-readahead-size = "2MB"
+
+[rocksdb.defaultcf]
+# The data block size. RocksDB compresses data based on the unit of block.
+# Similar to page in other databases, block is the smallest unit cached in block-cache.
+block-size = "64KB"
 
-Besides `block cache` and `write buffer`, the system memory is also occupied in the following scenarios:
+# The compression mode of each layer of RocksDB data. The optional values include no, snappy, zlib, 
+# bzip2, lz4, lz4hc, and zstd.
+# "no:no:lz4:lz4:lz4:zstd:zstd" indicates there is no compression of level0 and level1; lz4 compression algorithm is used
+# from level2 to level4; zstd compression algorithm is used from level5 to level6.
+# no means no compression. lz4 is a compression algorithm whose speed and compression ratio are moderate. The 
+# compression ratio of zlib is high. It is friendly to the storage space, but its compression speed is slow. This
+# compression occupies many CPU resources. Different machines deploy compression modes according to CPU and I/O resources.
+# For example, if you use the compression mode of "no:no:lz4:lz4:lz4:zstd:zstd" and find much I/O pressure of the 
+# system (run the iostat command to find %util lasts 100%, or run the top command to find many iowaits) when writing
+# (importing) a lot of data while the CPU resources are adequate, you can compress level0 and level1 and exchange CPU
+# resources for I/O resources. If you use the compression mode of "no:no:lz4:lz4:lz4:zstd:zstd" and you find the I/O
+# pressure of the system is not big when writing a lot of data, but CPU resources are inadequate. Then run the top 
+# command and choose the -H option. If you find a lot of bg threads (namely the compression thread of RocksDB) are 
+# running, you can exchange I/O resources for CPU resources and change the compression mode to "no:no:no:lz4:lz4:zstd:zstd".
+# In a word, it aims at making maximal use of the existing resources of the system and giving full play to TiKV performance
+# in terms of the current resources.
+compression-per-level = ["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
+
+# The RocksDB memtable size
+write-buffer-size = "128MB"
+
+# The maximum number of the memtables. Before the data is written in RocksDB, it is recorded in WAL logs and then 
+# inserted in memtables. When the memtable size reaches the limiting size of write-buffer-size, the memtable will turn
+# into read only and generate a new memtable receiving new write operations. The flush threads of RocksDB will flush
```

Comment:
-> The data written into RocksDB is first recorded in the WAL log, and then inserted into memtables. When the memtable reaches the size limit of `write-buffer-size`, it turns into read only and generates a new memtable receiving new write operations.